src/ports/postgres/modules/deep_learning/madlib_keras_wrapper.py_in

```diff
@@ -199,7 +215,34 @@ def get_optimizers():
 def compile_model(model, compile_params):
     optimizers = get_optimizers()
     (opt_name,final_args,compile_dict) = parse_compile_params(compile_params)
-    optimizer = optimizers[opt_name](**final_args) if final_args else opt_name
-    model.compile(optimizer=optimizer,
-                  loss=compile_dict['loss'],
-                  metrics=compile_dict['metrics'])
+    compile_dict['optimizer'] = optimizers[opt_name](**final_args) if final_args else opt_name
+    validate_compile_params(compile_dict)
+    model.compile(**compile_dict)
+
+def validate_compile_params(compile_dict):
+    _assert(compile_dict['metrics'] is None or type(compile_dict['metrics']) is list,
+            "wrong input type for compile parameter metrics: multi-output model"
+            "is not supported yet, please pass a list")
```

Comment:
1. I am confused by the `multi-output model` error message. Can you explain?
1. Should both of these conditions throw the same error message ? This also applies to all the other asserts in this function
1. Keras allows user to pass in a custom metric but we can't support it. We should add this to the docs

Reply:
Keras accepts a dict as `metrics` if you are building a multi-output model (that we don't support yet).

Reply:
I see thanks.
But what if the user passes in just a string? The check will fail and we will print the wrong error message

Reply:
The keras documentation indicates that it should be a list. Taken from https://keras.io/models/model/
```
Typically you will use metrics=['accuracy']
```